fla/layers/mom.py

```
@@ -0,0 +1,823 @@
+            #     recurrent_state[0] = recurrent_state_
+            
+            # o_list = o_list[:, :, -max_len:]
+
+            # o = reconstruct(o_list, indices=indices, sorted_indices=sorted_indices, batch_size=batch_size,
+            #                 seq_len=seq_len, topk=self.topk, routing_weights=routing_weights, mask=mask)
+
+        o = o.squeeze(0).contiguous()
+        o = pad_input(o, indices_q, batch_size*self.num_memories, max_len)
+        o = rearrange(o, '(e b) l h d -> e b l (h d)', b=batch_size)
+        o = reconstruct(o, indices=indices, sorted_indices=sorted_indices, batch_size=batch_size,
+                seq_len=seq_len, topk=self.topk, routing_weights=routing_weights, mask=mask)
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)